htdocs/xoops_lib/modules/protector/library/HTMLPurifier/Zipper.php

```diff
@@ -33,7 +33,7 @@ public function __construct($front, $back) {
      * @param Array to zipper-ify.
      * @return Tuple of zipper and element of first position.
      */
-    static public function fromArray($array) {
+    public static function fromArray($array) {
```

Comment:
We should avoid changing vendor supplied code. Unless it is an actual error that is not fixed upstream, it should be left in its original form. Following this rule makes analyzing/auditing changes much easier.